### PR TITLE
Bump the dependency version of Omniauth to 2.0 and bump major version of the strategy to 1.0.0

### PR DESCRIPTION
**Pull request by marcbest**
https://github.com/omniauth/omniauth/wiki/Upgrading-to-2.0#rails

### CODE REVIEW

File: lib/omniauth/strategies/strava.rb [hunk 1]
```diff
@@ -12,6 +12,13 @@ class Strava < OmniAuth::Strategies::OAuth2
       }
       option :scope, 'read'
 
+      def token_params
```
Comment (marcbest) [inline lib/omniauth/strategies/strava.rb:15]: I am not sure why this was required, but it doesnt seem like the `client_id` and the `client_secret` were being added to the request for the token

**Review by thomagos — APPROVED** [review]
Tested and works as expected.

**Comment by sethherr** [comment]
@thogg4 - can you merge this?

**Comment by thogg4** [comment]
Thanks @marcbest

**Comment by marcbest** [comment]
Thank you for merging @thogg4. If you have time would it please be possible to create a new release, happy to help out in any way 🙇 thanks!

**Comment by thogg4** [comment]
@marcbest Just cut one. Thanks for the contribution.

**Pull request merged**